incubator/java-microprofile/README.md

```diff
@@ -1,47 +1,63 @@
-# Java Microprofile
+# Java Microprofile Stack
 
-A stack is the foundation of your application which provides a prototype platform definition for Java Microprifile based projects.
+The Java Microprofile stack provides a consistent way of developing microservices based upon the [Eclipse MicroProfile® specifications](https://microprofile.io). This stack lets you use [Maven](https://maven.apache.org) to develop applications for [Open Liberty](https://openliberty.io) runtime, which includes OpenJDK with container-optimizations in OpenJ9.
```

Comment:
...which includes OpenJDK makes it sound like that is installed as part of Open Liberty is that the intent?

Reply:
might be best to say `running on`

Reply:
addressed